[TH]_Psycho_Pass_THE_MOVIE.ass

```diff
@@ -684,105 +692,132 @@ Dialogue: 0,1:01:04.99,1:01:07.83,Default,,0,0,0,,He joined forces with Sybil
+Dialogue: 0,1:04:08.47,1:04:11.26,Default,,0,0,0,,พวกเค้ามีอุปกรณ์เจาะระบบขั้นสูง
+<<<<<<< HEAD
+Dialogue: 0,1:04:11.39,1:04:15.60,Default,,0,0,0,,ตลอดจนสามารถติดต่อกับพ่อค้าตลอดมืดได้อีกด้วยค่ะ
+=======
+Dialogue: 0,1:04:11.39,1:04:15.60,Default,,0,0,0,,ตลอดจนเครื่องมือสื่อสารที่จำเป็นเพื่อติดต่อกับเขตตะวันตก
+>>>>>>> origin/Sentence-Line-661-759
+Dialogue: 0,1:04:17.06,1:04:19.23,Default,,0,0,0,,ไม่คิดว่ามันแปลกๆ เหรอคะ ?
+Dialogue: 0,1:04:20.27,1:04:21.48,Default,,0,0,0,,น่าจะเป็นอย่างงั้นนะ
+Dialogue: 0,1:04:21.56,1:04:25.23,Default,,0,0,0,,จะคิดว่ามีใครบงการพวกนั้นอยู่ก็ได้เหมือนกัน
+Dialogue: 0,1:04:27.23,1:04:28.28,Default,,0,0,0,,โควกามิ
+Dialogue: 0,1:04:30.53,1:04:31.74,Default,,0,0,0,,ดูเหมือนเราทั้งคู่
+Dialogue: 0,1:04:32.41,1:04:36.66,Default,,0,0,0,,จะทำเรื่องที่เกินตัวที่เราจะทำได้ซะแล้วสิ
+Dialogue: 0,1:04:41.83,1:04:43.58,Default,,0,0,0,,คืนนี้พักที่นี่เถอะ
+Dialogue: 0,1:04:43.79,1:04:44.79,Default,,0,0,0,,พรุ่งนี้เช้า
+Dialogue: 0,1:04:44.96,1:04:47.63,Default,,0,0,0,,เราจะพาเธอไปยังจุดที่เดินไปเกาะชัมบาล่าได้
+Dialogue: 0,1:04:47.76,1:04:51.18,Default,,0,0,0,,ถ้าเธออยู่เงียบๆ จะไม่มีอันตรายใดๆ เกิดขึ้นกับเธอ
+Dialogue: 0,1:04:51.84,1:04:54.85,Default,,0,0,0,,มันจะเป็นการมากไปรึเปล่า ถ้าขอให้คุณยอมมอบตัวน่ะค่ะ ?
+Dialogue: 0,1:04:56.10,1:04:57.26,Default,,0,0,0,,ขอร้องล่ะ
+Dialogue: 0,1:04:57.39,1:05:00.35,Default,,0,0,0,,อย่าบอกเซมหรือคนอื่นว่าเธอคือเจ้าหน้าที่สังเกตการณ์
+Dialogue: 0,1:05:00.43,1:05:02.10,Default,,0,0,0,,ถ้าพวกนั้นรู้ว่าเธอเกี่ยวข้องกับรัฐบาล
+Dialogue: 0,1:05:02.27,1:05:04.94,Default,,0,0,0,,พวกนั้นคงจะทำเป็นไม่สนใจไม่ได้
+Dialogue: 0,1:06:11.84,1:06:14.47,Default,,0,0,0,,เฮ้ย มันเลยเวลาแล้วนะ
+Dialogue: 0,1:06:15.30,1:06:16.47,Default,,0,0,0,,ทุกอย่างเรียบร้อย
+Dialogue: 0,1:06:22.35,1:06:23.56,Default,,0,0,0,,เคลื่อนพลได้
+Dialogue: 0,1:06:24.02,1:06:26.02,Default,,0,0,0,,รับทราบ
+Dialogue: 0,1:06:25.02,1:06:26.02,Default,,0,0,0,,รับทราบ
+Dialogue: 0,1:07:10.56,1:07:12.65,Default,,0,0,0,,ผู้ก่อการร้ายที่มาที่ญี่ปุ่น
+Dialogue: 0,1:07:12.78,1:07:15.44,Default,,0,0,0,,มีหนังสือแปลกๆ มากับตัวด้วยค่ะ
+Dialogue: 0,1:07:15.90,1:07:19.37,Default,,0,0,0,,หนังสือของพรูสท์ In Search of Lost Time
+Dialogue: 0,1:07:19.62,1:07:21.95,Default,,0,0,0,,นั่นน่าจะเป็นของฉัน
+Dialogue: 0,1:07:22.54,1:07:24.75,Default,,0,0,0,,ตอนที่แซมลินยังอยู่ที่นี่
+Dialogue: 0,1:07:24.91,1:07:27.12,Default,,0,0,0,,เค้าชอบมากวนฉันว่าอยากอ่านมันมาก
+Dialogue: 0,1:07:27.37,1:07:32.17,Default,,0,0,0,,ฉันคิดว่าฉันทำหายไป แต่เค้าคงเอาไปนั่นแหละ
+Dialogue: 0,1:07:32.59,1:07:34.92,Default,,0,0,0,,ชายที่ชื่อเซม
+Dialogue: 0,1:07:35.05,1:07:37.68,Default,,0,0,0,,เค้าบอกฉันว่า คุณชี้ทางสว่างให้เค้า
+Dialogue: 0,1:07:38.26,1:07:41.60,Default,,0,0,0,,คุณโควกามิ ไม่ได้เป็นแค่ที่ปรึกษาทางการรบธรรมดา
+Dialogue: 0,1:07:41.72,1:07:45.06,Default,,0,0,0,,แต่กลายเป็น ผู้นำลัทธิให้คนพวกนี้ไปแล้วไม่ใช่เหรอคะ ?
+Dialogue: 0,1:07:46.81,1:07:50.60,Default,,0,0,0,,ถ้าให้พูดตรงๆ ฉันคิดว่ามันไม่ถูกต้อง และมันก็ไม่เข้ากับฉันเลย
```

Comment:
ขอแก้ไขนะครับ sit right ไม่ได้แปลว่าถูกต้อง
จากซับ eng คิดว่าน่าจะประมาณนี้มากกว่า "พูดตามตรง ฉันก็ไม่เข้าใจเหมือนกัน / ฉันเองก็ไม่อยากยอมรับ และมันก็ไม่เข้ากับฉันเลย"
sit right with = be acceptable or understandable to someone.
อ้างอิง http://idioms.thefreedictionary.com/sit+right+with